OneLogin.xcodeproj/project.pbxproj

```diff
@@ -2194,8 +2194,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/govuk-one-login/mobile-ios-secure-store";
 			requirement = {
-				kind = upToNextMajorVersion;
-				minimumVersion = 1.0.0;
+				branch = "feature/dcmaw-8006-use-passcode";
```

Comment:
Should this be getting committed?

Reply:
It was only there while the PR on SecureStore was awaiting approval/being merged in case yourself or Dylan wanted to test this branch with that change while we waited. It's been reverted to point to the main branch now 🙂